src/app.tsx

```diff
@@ -295,29 +296,37 @@ export default function Chat() {
 
         {/* Input Area */}
         <form
-          onSubmit={(e) =>
+          onSubmit={(e) => {
+            e.preventDefault();
             handleAgentSubmit(e, {
               data: {
                 annotations: {
                   hello: "world",
                 },
               },
-            })
-          }
-          className="p-3 bg-input-background absolute bottom-0 left-0 right-0 z-10 border-t border-neutral-300 dark:border-neutral-800"
+            });
+            setTextareaHeight("auto"); // Reset height after submission
+          }}
+          className="p-3 bg-gray-50 absolute bottom-0 left-0 right-0 z-10 border-t border-neutral-300 dark:border-neutral-800 dark:bg-neutral-900"
         >
           <div className="flex items-center gap-2">
             <div className="flex-1 relative">
-              <Input
+              <Textarea
                 disabled={pendingToolCallConfirmation}
                 placeholder={
                   pendingToolCallConfirmation
                     ? "Please respond to the tool confirmation above..."
-                    : "Type your message..."
+                    : "Send a message..."
                 }
-                className="pl-4 pr-10 py-2 w-full rounded-full"
+                className="flex w-full border border-neutral-200 dark:border-zinc-700 px-3 py-2 text-base ring-offset-background placeholder:text-neutral-500 dark:placeholder:text-neutral-400 focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-neutral-300 dark:focus-visible:ring-neutral-700 focus-visible:ring-offset-2 dark:focus-visible:ring-offset-neutral-900 disabled:cursor-not-allowed disabled:opacity-50 md:text-sm min-h-[24px] max-h-[calc(75dvh)] overflow-hidden resize-none rounded-2xl !text-base pb-10 dark:bg-neutral-900"
```

Comment:
Same here, the `dark:border-zinc-700` is now using `zinc` instead of `gray` which you use above and instead of `neutral` which we preferred for this repo